Sources/SwiftParser/Specifiers.swift

```diff
@@ -415,7 +415,120 @@ extension RawAccessorEffectSpecifiersSyntax: RawEffectSpecifiersTrait {
       }
     }
   }
+}
+
+extension RawDeinitEffectSpecifiersSyntax {
+  enum MisspelledAsyncSpecifiers: TokenSpecSet {
```

Comment:
Oh, I just noticed that `MisspelledAsyncSpecifiers` is inconsistently named with `MisspelledThrowsTokenKinds`. I’m fixing that in https://github.com/apple/swift-syntax/pull/1767. You’ll probably need to rebase once that PR is merged.

Reply:
Updated